.github/actions/spelling/patterns.txt

```diff
@@ -0,0 +1,107 @@
+# This file contains various spellcheck patterns for @check-spelling/check-spelling
+# and is used in the spellcheck workflow. See ../../workflows/spellcheck.yml for more information.
+
+# marker to ignore all code on line
+^.*/\* # no-spell-check-line \*/.*$
+# marker for ignoring a comment to the end of the line
+// # no-spell-check.*$
+# marker for ignoring all lines with disabled spellcheck
+.*spellcheck: disable
+
+# git index header
+index [0-9a-z]{7,40}\.\.[0-9a-z]{7,40}
+
+# data urls
+(['"])data:.*?\g{-1}
```

Comment:
Same thing here:
```suggestion
(['"])data:*?\g{-1}
```

Reply:
Don't mess with that. That's from the original `patterns.txt` template.